gather_istio

```diff
@@ -133,11 +140,11 @@ function getEnvoyConfigForPodsInNamespace() {
       local logPath=${BASE_COLLECTION_PATH}/namespaces/${podNamespace}/pods/${podName}
       mkdir -p ${logPath}
 
-      echo "Pilot config for pod ${podName}.${podNamespace} from istiod ${pilotName}.${controlPlaneNamespace}" 2>&1 1>${logPath}/pilotConfiguration
-      oc exec ${pilotName} -n ${controlPlaneNamespace} -c discovery -- bash -c "/usr/local/bin/pilot-discovery request GET /debug/config_dump?proxyID=${podName}.${podNamespace}" 2>&1 1>${logPath}/pilotConfiguration
+      echo "Pilot config for pod ${podName}.${podNamespace} from istiod ${pilotName}.${controlPlaneNamespace}" > ${logPath}/pilotConfiguration 2>&1
```

Comment:
nit: Strictly speaking `2>&1` is not necessary, I believe `echo` will never fail.

Reply:
Lol. At least if it does, there are likely bigger issues.